web-nfc/index.js

```diff
@@ -6,7 +6,7 @@ scanButton.addEventListener("click", async () => {
     await reader.scan();
     log("> Scan started");
 
-    reader.addEventListener("error", () => {
+    reader.addEventListener("error", (error) => {
```

Comment:
Thanks for catching this typo!
I'd suggest though we use `event` instead of `error`

```suggestion
    reader.addEventListener("error", (event) => {
```
And use ```log(`Argh! ${event.message}`);```

Reply:
Done.